### ISSUE TITLE
Deprecated Buffer usage

### ISSUE THREAD

**Issue by oliversalzburg** [brianc/node-postgres#1811]
The module still depends on a version of `packet-reader` that has deprecated Buffer usage.

```
consoleStream.js:43
(node:44680) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
warning.js:18
    at showFlaggedDeprecation (buffer.js:160:11)
    at new Buffer (buffer.js:175:3)
    at module.exports.Reader.addChunk (.packages\packet-reader@f4fc956ec61b187cd18790c0b0a6fa5d1d3a30de4a5493e3277460485ed188f2\index.js:35:21)
    at Socket.<anonymous> (.packages\pg@1070266975c840c14642630da3a4733f2608c038eac5d97764832e1dff16eccb\lib\connection.js:116:18)
```

The PR to fix this has been open for 2 months. https://github.com/brianc/node-packet-reader/pull/5

**Comment by bitorb** [comment]
@brianc @oliversalzburg Any plan about this issue? The warning really is annoying. Thanks

**Comment by oliversalzburg** [comment]
I think I have already been as annoying as possible about this issue :(

**Comment by brianc** [comment]
I should have a fix for this today.

**Comment by brianc** [comment]
Alright I pushed the fix out...but my terminal is still showing the old version as the only one available in npm:

```
~/src/node-postgres [bmc/upgrade-packet-reader !]$ npm info packet-reader

packet-reader@0.3.1 | MIT | deps: none | versions: 6
Read binary packets...
https://github.com/brianc/node-packet-reader

dist
.tarball: https://registry.npmjs.org/packet-reader/-/packet-reader-0.3.1.tgz
.shasum: cd62e60af8d7fea8a705ec4ff990871c46871f27

maintainers:
- brianc <brian.m.carlson@gmail.com>

dist-tags:
latest: 0.3.1  

published a year ago by brianc <brian.m.carlson@gmail.com>
```

where as on npmjs.com we get:

![image](https://user-images.githubusercontent.com/50081/53105490-d7e1a800-34f6-11e9-9bf3-5f98d8a3dcf6.png)


A bit confused as to what could be causing the mis-match...

**Comment by sehrope** [comment]
I see the same older version as well. There were some delays with NPM publishing earlier this month (https://status.npmjs.org/incidents/hry8fvybglkw). Maybe they're having similar replication issues today as well.

**Comment by brianc** [comment]
ah thanks for checking - I'll wait a few hours & see if it clears up.

**Comment by brianc** [comment]
published a fix `pg@7.8.1` - please lemme know if you hit any other deprecation warnings! thanks.

**Comment by oliversalzburg** [comment]
Everything looks clean here. Thanks for taking care of it :)
